src/requests.rs

```diff
@@ -219,8 +219,14 @@ pub async fn fetch_proof_from_l1(
         })
         .unwrap();
 
+    if prove_tx.is_none() {
+        let msg = format!("Proof doesn't exist for batch {} on network {}, exiting...", batch_number.to_string().red(), network.red());
```

Comment:
I'd also mention - that if this is 'mainnet' - then the proof will arrive soon, and to retry. And if this is testnet, then we don't generate proofs for 100% of blocks there (I think).